homeassistant/helpers/llm.py

```diff
@@ -170,6 +169,15 @@ class AssistAPI(API):
         INTENT_GET_TEMPERATURE,
     }
 
+    def __init__(self, hass: HomeAssistant) -> None:
+        """Init the class."""
+        super().__init__(
+            hass=hass,
+            id=LLM_API_ASSIST,
+            name="Assist",
+            prompt_template="Call the intent tools to control the system. Just pass the name to the intent.",
```

Comment:
Please break long strings around max 88 characters per line.

Reply:
Addressed feedback in https://github.com/home-assistant/core/pull/117873